assets/app/views/_project-page.html

```diff
@@ -1,12 +1,18 @@
 <project-nav></project-nav>
 <div class="wrap">
-  <sidebar></sidebar>
-
-  <div class="container-fluid container-main">
-    <div class="row">
-      <div class="col-sm-12 col-md-10" ng-transclude>
+  <div class="sidebar-left">
+    <sidebar></sidebar>
+  </div>
+  <div class="container-main">
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-md-12" ng-transclude>
 
+        </div>
       </div>
     </div>
   </div>
+  <div class="sidebar-right">
```

Comment:
It's set to display none by default and only shows when body class="show-sidebar-right" is applied.